playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -69,10 +69,11 @@
     - skip_ansible_lint
     - update_airship_osh_site
 
-- name: Copy site to Shipyard
+- name: Copy shipyard to expected location
```

Comment:
Technically we are copying aggregated and partially validated site document to shipyard for publishing to deckhand.
So previous text is more closer.